js/dfpslotsprovider.js

```diff
@@ -214,7 +219,7 @@ export default class DFPSlotsProvider extends React.Component {
   }
 
   render() {
-    const children = <div> {this.props.children} </div>;
+    const children = <React.Fragment> {this.props.children} </React.Fragment>;
```

Comment:
I really like `<Fragment>`, but this component was introduced in ~16.X, react-dfp on the other hand is still supporting older versions like react 15.0 and 14.0 so this will not work, could u please revert this change?